.circleci/config.yml

```diff
@@ -83,34 +106,74 @@ jobs:
             python3 --version
             pip3 install pyyaml
 
-      - checkout
-      - run:
-          name: Checkout vsi_common
-          command: |
-            git clone --recursive https://github.com/VisionSystemsInc/vsi_common.git /vsi
-
+      - checkout_in_vsi_common
       - setup_remote_docker
 
       - run:
           name: Additional setup
           command: |
-            sed -i 's|context: ../..|context: ${VSI_COMMON_DIR}|' docker-compose.yml
             docker login -u "${DOCKER_USER}" -p "${DOCKER_PASS}"
 
       - ci_load:
           step_name: Build recipes (ci_load)
-          compose_file: ./docker-compose.yml
+          compose_file: ${RECIPE_DIR}/docker-compose.yml
 
       - ci_load:
           step_name: Build tests (ci_load)
-          compose_file: ./tests/docker-compose.yml
+          compose_file: ${RECIPE_DIR}/tests/docker-compose.yml
+
+      - run:
+          name: Run integration tests
+          command: |
+            source setup.env
+            just test recipe
+
+
+  # test selected scripts for windows compatability
+  windows_test:
+    executor:
+      name: win/default
+      shell: bash.exe
+    working_directory: "C:/Users/circleci/vsi"
+    environment:
+      VSI_COMMON_DIR: "C:/Users/circleci/vsi"
+      RECIPE_DIR: "C:/Users/circleci/vsi/docker/recipes"
+
+    steps:
+      - run: systeminfo
+      # - run: printenv
+
+      - run:
+          name: Update bash environment
+          command: |
+
+            # Give preference to bash utilities via PATH.
+            # For example, prefer "C:/Program Files/Git/usr/bin/find" over
+            # the windows executable "C:/Windows/system32/find"
+            echo 'export PATH="/c/Program Files/Git/usr/bin":${PATH}' >> "${BASH_ENV}"
+
+            # setting an alias for find did not work command did not seem to work
+            # echo 'alias find="/c/Program Files/Git/usr/bin/find.exe"' >> "${BASH_ENV}"
```

Comment:
Yes, I would avoid using alias even if they did work, because you could easily find yourself missing yet another command that needs an alias, etc...